metrotranscom/doorway
# Add Landing page content

# Pull Request Template

## Issue Overview
Add Landing Page Content and ES translations

This PR addresses #issue


## Description
Add Landing page content from team shown in [this sheet](https://docs.google.com/spreadsheets/d/1Cr3L2H2iLWMS1qNOYYsnLJCrB1RNo_PXQiirzfg38PE/).  The language is also shown in the Figma Landing page mocks, but the source of truth is the sheet.

We've only received ES & ZH translations back so far, so going to submit this PR now to unblock progress. 
![image](https://github.com/metrotranscom/doorway/assets/12400715/afc6736a-75a6-4809-b632-2cc4a96c6100)
![image](https://github.com/metrotranscom/doorway/assets/12400715/5e0b3de1-7319-4457-9cb5-a5b224408e00)

Still need to:
-  add 2 other language translations

## How Can This Be Tested/Reviewed?
`yarn dev` on this branch and look at the Landing page. Test that the links are working and that all the content looks similar to the Figma mocks (or the screenshots included here). Switch languages and ensure no translations are missing (ES & ZH only)
![image](https://github.com/metrotranscom/doorway/assets/12400715/38166f6b-461a-46ed-8e27-afa3bb80ae19)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
